src/include/network/lan/udev.rb

```diff
@@ -45,6 +45,28 @@ def update_udev_rule_key(rule, key, value)
       rule
     end
 
+    # Writes new persistent udev net rules and tells udevd to update its configuration
+    def write_update_udevd(udev_rules)
+      SCR.Write(path(".udev_persistent.rules"), udev_rules)
+      SCR.Write(path(".udev_persistent.nil"), [])
+
+      update_udevd
+    end
+
+    # Tells udevd to reload and update its configuration
+    def update_udevd
+      SCR.Execute(path(".target.bash"), "udevadm control --reload")
+
+      # When configuring a new s390 card, we neglect to fill
+      # its Items[i, "udev", "net"], causing jumbled names (bnc#721520)
+      # The udev trigger will make udev write the persistent names
+      # (which it already has done, but we have overwritten them now).
+      SCR.Execute(
+        path(".target.bash"),
+        "udevadm trigger --subsystem-match=net --action=add"
+      ) == 0
```

Comment:
Compared to the old code there is an added `== 0` test. But the behavior is not documented in the comment for the new(!) method. It should say `# @return void` as it is actually used in the new code or `# @return [Boolean] we succeeded in tickling the daemon or whatever` as it is implemented.